src/orquestra/sdk/_base/abc.py

```diff
@@ -145,6 +145,17 @@ def list_workflow_runs(
         """
         raise NotImplementedError()
 
+    # TODO: 852 change to WorkflowLogs?
```

Comment:
I think we should do this todo now, if not this ticket number is for the ticket this PR is supposed to close.